.github/workflows/dev-astro.yml

```diff
@@ -35,6 +39,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
```

Comment:
Don't you want to get rid of the on push for the release branch we are no longer working on?

Reply:
oh yeah, good idea.